src/Stripe.net/Services/Sources/SourceThreeDSecureCreateOptions.cs

```diff
@@ -1,13 +1,15 @@
 namespace Stripe
 {
     using Newtonsoft.Json;
+    using Stripe.Infrastructure;
 
     public class SourceThreeDSecureCreateOptions : INestedOptions
     {
         [JsonProperty("customer")]
         public string Customer { get; set; }
 
         [JsonProperty("card")]
+        [AllowNameMismatch]
         public string CardId { get; set; }
```

Comment:
Why not rename to `Card` here?

Reply:
🤷‍♂  It's 3DS and you didn't change it in #1738 so I assumed you just didn't want to change the API. But I'm happy to change it if you think it's preferable.

Reply:
Yeah this one is worth changing, I must have missed it. It's on Sources which is still used so if you are willing to take that's likely better

Reply:
👍  Fixed in #1805.